packages/e2e.test.foundry-sdk-generator/src/legacy-client/loadObjects.test.ts

```diff
@@ -123,7 +122,7 @@ describe("LoadObjects", () => {
 
     const emp = assertOkOrThrow(result);
 
-    expectTypeOf(emp).toEqualTypeOf<{
+    expectTypeOf(emp).branded.toEqualTypeOf<{
```

Comment:
Because these were not actually being source code verified before (see below) this wasn't correct. `__apiName` became a branded string awhile ago. Which means these types arent TECHNICALLY correct. `.branded` looses things up seemingly stripping away brands allowing this check to keep working